src/RequestsLibrary/RequestsKeywords.py

```diff
@@ -34,6 +34,8 @@ def _common_request(
 
         self._capture_output()
 
+        kwargs.update({'verify': session.verify})
```

Comment:
This will break session less keywords (session is not defined in this case), and should not be necessary when the session has verify=False